package.json

```diff
@@ -65,65 +65,72 @@
     "@reach/tabs": "^0.18.0",
     "@remark-embedder/core": "^3.0.1",
     "@remark-embedder/transformer-oembed": "^3.0.0",
-    "@remix-run/express": "^1.16.0",
-    "@remix-run/node": "^1.16.0",
-    "@remix-run/react": "^1.16.0",
-    "@sentry/node": "^7.52.0",
-    "@sindresorhus/slugify": "^2.2.0",
+    "@remix-run/css-bundle": "^1.17.1",
+    "@remix-run/express": "^1.17.1",
+    "@remix-run/node": "^1.17.1",
+    "@remix-run/react": "^1.17.1",
+    "@remix-run/router": "^1.6.3",
+    "@remix-run/server-runtime": "^1.17.1",
+    "@sentry/node": "^7.55.2",
+    "@sindresorhus/slugify": "^2.2.1",
     "@tailwindcss/aspect-ratio": "^0.4.2",
     "@tailwindcss/typography": "^0.5.9",
     "@xstate/react": "^3.2.2",
     "bcrypt": "^5.1.0",
-    "better-sqlite3": "^8.3.0",
-    "cachified": "^3.5.3",
+    "better-sqlite3": "^8.4.0",
+    "cachified": "^3.5.4",
     "chalk": "^5.2.0",
+    "chokidar": "^3.5.3",
+    "close-with-grace": "^1.2.0",
     "cloudinary-build-url": "^0.2.4",
     "clsx": "^1.2.1",
     "compression": "^1.7.4",
     "cross-env": "^7.0.3",
     "date-fns": "^2.30.0",
-    "dotenv": "^16.0.3",
+    "dotenv": "^16.3.0",
     "emoji-regex": "^10.2.1",
     "error-stack-parser": "^2.1.4",
     "esbuild": "^0.17.19",
+    "execa": "^7.1.1",
     "express": "^4.18.2",
     "express-async-errors": "^3.1.1",
     "express-http-proxy": "^1.6.3",
-    "framer-motion": "^10.12.11",
+    "framer-motion": "^10.12.16",
     "fs-extra": "^11.1.1",
-    "gsap": "^3.11.5",
+    "gsap": "^3.12.1",
     "hast-util-to-html": "^8.0.4",
     "hast-util-to-string": "^2.0.0",
     "helmet": "^7.0.0",
     "isbot": "^3.6.10",
     "litefs-js": "^1.1.2",
     "lodash": "^4.17.21",
-    "lru-cache": "^7.14.1",
+    "lru-cache": "^10.0.0",
     "match-sorter": "^6.3.1",
     "md5-hash": "^1.0.1",
     "mdast-util-to-hast": "^12.3.0",
     "mdx-bundler": "^9.2.1",
-    "metascraper": "^5.34.4",
-    "metascraper-description": "^5.34.4",
-    "metascraper-image": "^5.34.4",
-    "metascraper-title": "^5.34.4",
+    "metascraper": "^5.34.7",
+    "metascraper-description": "^5.34.7",
+    "metascraper-image": "^5.34.7",
+    "metascraper-title": "^5.34.7",
     "morgan": "^1.10.0",
+    "node-fetch": "^3.3.1",
```

Comment:
Why was this dependency added?

Reply:
`generate-build-info.js`
https://github.com/kentcdodds/kentcdodds.com/blob/e755dfbef26b9beb21616133fc4c86fb613bbd51/other/generate-build-info.js#L1-L5